### PR TITLE
In insert_fieldset, remove html default "value" from input type="text"

### PR DESCRIPTION
**Pull request by umjennifer**
- In insert_fieldset, remove html default "value" from input type="text" (we don't have default values for varchars in MySQL)

### CODE REVIEW

**Comment by nolandseigler** [comment]
I think we should take a look at handling errors this way: https://fastapi.tiangolo.com/tutorial/handling-errors/#override-the-httpexception-error-handler instead of manually redirecting to error pages.

**Comment by umjennifer** [comment]
> I think we should take a look at handling errors this way: https://fastapi.tiangolo.com/tutorial/handling-errors/#override-the-httpexception-error-handler instead of manually redirecting to error pages.

Thanks for the link. I think we can remove all the error related changes in this PR and after we implement and merge CREATE for the rest of the entities, we can create a new PR solely focused on handling errors

**Review by nolandseigler — APPROVED** [review]

**Pull request merged**
